bootstrap/shell/shell.go

```diff
@@ -337,7 +337,14 @@ func (s *Shell) RunScript(ctx context.Context, path string, extra *env.Environme
 		args = []string{"-file", path}
 
 	case !isWindows && isBash:
-		command = "/bin/bash"
+		bashPath, err := s.AbsolutePath("bash")
+		if err != nil {
+			return fmt.Errorf("Error finding bash, needed to run scripts: %v.", err)
+		}
+		if s.Debug {
+			s.Commentf("Found bash at %s", bashPath)
+		}
```

Comment:
I wonder if this is too verbose/trivial even for debug logging?
I'm unsure, no strong feels, happy to leave it if it's useful / consistent with other debug logging.

Reply:
I added it because it mirrors the debug output we have a few lines up for windows. We could remove it for both though?

Reply:
I think the windows ones advertise bash or powershell because they diverge. I don't think we need to report finding bash in here.